javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -151,6 +151,45 @@ console.log(theHobbit.info());
 
 Before we go much further, there's something important you need to understand about JavaScript objects. All objects in JavaScript have a `prototype`. Stated simply, the prototype is another object that the original object _inherits_ from, which is to say, the original object has access to all of its prototype's methods and properties.
 
+To elaborate, there are two kinds of `prototype`, which are intertwined with each other. The _first_ kind is the one that exists on a **function**, which is referred to as `Player.prototype` in the case of the `Player` function in the previous example. 
```

Comment:
I think I get what you're going for in saying there are two "kinds" of prototype, but I feel like the way things are phrased isn't *quite* accurate. There's only one "kind" of prototype, the actual prototype of something which in the spec is referred to as `[[prototype]]`, but there's also the `.prototype` _property_ which behaves in a similar kind of way but has distinctly different rules as to what that property is.

I suppose I'm just not sure about referring to this as two "kinds" of prototype, since really one is the prototype, and the other is the value of the `.prototype` property, if that makes sense.

Do you think there might be a way you can reword this to make that differentiation more clear?